### PR TITLE
chore: prettify code

### PR DESCRIPTION
**Pull request by dryajov**
just some quick cleanup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,39 +145,36 @@ Both for dialing and listening.
...
+||           |      |    |      |                ||                                     |   |  |             | |   | 
+||libp2p-tcp |libp2p-ws  | .... |libp2p-circuit  ||  listener handles STOP messages from|   |  | listener    | |   | 
+||           |      +-------------------------------------------------------------------------->             | |   | 
+||           |           |      |pluggs in just  ||  circuit-relay nodes                |   |  +-------------+ |   | 
+||           |           |      |as any other    ||                                     |   |                  |   | 
+||           |           |      |transport       ||                                     |   +------------------+   | 
+|+-----------------------------------------------+|                                     |                          | 
+|                                                 |                                     |                          | 
+|                                                 |                                     |                          | 
++-------------------------------------------------+                                     +--------------------------+
```
Comment (daviddias) [inline README.md]: You know you can make smaller boxes, right? :)

**Comment by dryajov** [comment]
They're proportional ;)

screen shot 2018-02-07 at 9 49 44 am

**Comment by daviddias** [comment]
@dryajov this one has merge conflicts and fails CI. Are you on top of it?

**Comment by dryajov** [comment]
yep - I'm on it!

**Comment by dryajov** [comment]
added more coverage

**Comment by daviddias** [comment]
@dryajov seems that coveralls is not enabled. @VictorBjelkholm ?

**Comment by dryajov** [comment]
I've updated circle's yaml for coveralls but getting this
screen shot 2018-03-21 at 1 00 30 pm

**Comment by daviddias** [comment]
it needs to get the token set.

**Comment by daviddias** [comment]
I've just set the token. Let's see it run :)

**Review by daviddias — APPROVED** [review]

**Comment by dryajov** [comment]
Coverage is still a bit low at ~75 and I want to get it up to ~85. But as far as the cleanup goes, I think this done.

**Comment by daviddias** [comment]
@dryajov have you triple checked that all of these changes do not break anything?

**Comment by dryajov** [comment]
I've double checked, but I'll triple check - hang on.

**Review by jacobheun — APPROVED** [review]
I ran the rebased version of this against libp2p and js-ipfs tests. It looks good!

**Pull request merged**